docs/OBOOpsLogistics.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc
+title: OBO Operations Committee Logistics
```

Comment:
Note that this page needs to be added to the sidebar in place of these:

- [Meetings](https://obofoundry.org/docs/CommitteeMeetings.html)
- [Mailing Lists](https://obofoundry.org/docs/MailingLists.html)
- [OBO Calendar](https://obofoundry.org/docs/OBOCalendar.html)

I will do that in a separate PR